src/components/ProjectDetailsModal.js

```diff
@@ -14,19 +17,35 @@ class ProjectDetailsModal extends Component {
       var url = this.props.data.url;
       if (this.props.data.technologies) {
         var tech = technologies.map((icons, i) => {
-          return (
-            <li className="list-inline-item mx-3" key={i}>
-              <span>
-                <div className="text-center">
-                  <i className={icons.class} style={{ fontSize: "300%" }}>
-                    <p className="text-center" style={{ fontSize: "30%" }}>
-                      {icons.name}
-                    </p>
-                  </i>
-                </div>
-              </span>
-            </li>
-          );
+          // For using icon from iconify
+          if (icons.name === "api"){
```

Comment:
smells of duplication :) I think, in future we should consider making these their own components

Reply:
You mean when the next case like this is needed?